utils/check_copies.py

```diff
@@ -238,11 +247,10 @@ def blackify(code: str) -> str:
     """
     has_indent = len(get_indent(code)) > 0
     if has_indent:
+        pass
```

Comment:
pass will still run the nex line , right?

Reply:
Yes, but I'll just remove the pass it's useless